src/assets/stylesheets/theme/fonts.scss

```diff
@@ -1,6 +1,28 @@
 /* fonts
 ================================================== */
 
+/* FiraSans-Light 300 */
+@font-face {
+  font-family: 'Fira Sans';
+  font-display: swap;
+  font-style: normal;
+  font-weight: 300;
+  src:
+    local(''),
```

Comment:
FMI: What is this `local('')` for? Which effect does it have?

Reply:
It was there before. I think it is about not to use a local font. 
Please see here..

https://stackoverflow.com/questions/3837249/font-face-src-local-how-to-use-the-local-font-if-the-user-already-has-it
https://www.paulirish.com/2009/bulletproof-font-face-implementation-syntax/

Reply:
I would leave it as it is for now.